### ISSUE TITLE
[Lead Space Block]: QA: Heading level is not in an hierarchical order

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9858]
### Description

Page Element:
heading level order

Issue Description:
Heading level is not in an hierarchical order. found the heading level order as h1,h2 and h4 instead of H1, H2, H3 …

Recommend Fix:
As an accessibility best practice, Heading level should be in hierarchical order. Heading level should not be skipped. 

User Impact:
If page headings are not in the right sequence, screen reader users find it difficult to comprehend the hierarchical structure of the page content. 

Test Tool:
JAWS FF
Voiceover safari iPhone

### Component(s) impacted

Lead space Block
1. Default
2. with video
(web component canary and web component staging)

### Browser

Safari, Firefox

### Carbon for IBM.com version

v1.40.0-rc.0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

Web component Canary https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components Web component staging https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
